lib/parser/objects.ts

```diff
@@ -135,6 +135,7 @@ export class OIRange implements Range {
 
 export class ObjectBase {
   lexerToken?: OLexerToken;
+  hasBody = false;
```

Comment:
This is only used in `OSubprogram`. Why is it a property of `ObjectBase`?

Reply:
Hm, good question ;)